packages/beacon-node/test/unit/api/impl/validator/duties/proposer.test.ts

```diff
@@ -21,7 +21,7 @@ import {zeroProtoBlock} from "../../../../../utils/mocks/chain/chain.js";
 
 use(chaiAsPromised);
 
-describe("get proposers api impl", function () {
+describe.skip("get proposers api impl", function () {
```

Comment:
I'm sorry but this tests make no sense. Every time you change the implementation slightly they break because what they do is the following:

Given a function

```ts
function foo(obj) {
  obj.bar()
}
```
Assert that if you call `foo()`, `bar()` is called. What's the point of this? It's re-implementing the source code as a test.

For errors, it tests that after setting `objStub.bar.throws()`, calling `foo()` also throws. Again I don't see what return on investment this test brings to Lodestar for the goals of correctness and maintainability.

Reply:
Agree, hate those tests which just mirror internal implementation details